Cargo.toml

```diff
@@ -63,7 +63,7 @@ trillium-head = "0.2.1"
 trillium-opentelemetry = "0.4.0"
 trillium-router = "0.3.6"
 trillium-rustls = "0.4.2"
-trillium-testing = "0.5.0"
+trillium-testing = "0.5.4"
```

Comment:
This is necessary to make trillium-testing's TestConn not report closure when the body has been read to completion